app/actuators/ResultCard.js

```diff
@@ -151,7 +151,10 @@ export default class ResultCard extends Component {
 			const scrollFlag = scrollElement.scrollHeight > resultElement.clientHeight;
 			if (!flag && !scrollFlag && scrollElement && !this.props.pagination) {
 				const scrollHeight = resultElement.clientHeight - 100;
-				if (scrollHeight > 0) {
+				if (this.props.scrollOnTarget) {
+					scrollElement.style.height = "auto";
+					scrollElement.style.paddingBottom = 0;
```

Comment:
No need to specify the paddingBottom here

Reply:
@metagrover Updated.